scripts/gen_binding.py

```diff
@@ -313,11 +313,11 @@ def normalize_arg(arg):
     returns: {ffi_ret}
   }},"""
 
-    js_impl_full = "\n".join(js_impl)
+    js_impl_full = "\n".join(js_impl + (['this.cleanUp()'] if methods_only else []))
```

Comment:
lets keep this change separate from the PR for now :)

Reply:
Will revert in the next commit so that user has to explicitly call this.cleanUp() for the time being until we've better determined whether it makes sense to automate this process and, if so, what the best approach to do so is.